mods/gen4/formats-data.js

```diff
@@ -660,7 +660,7 @@ exports.BattleFormatsData = {
 		tier: "UU",
 	},
 	glaceon: {
-		randomBattleMoves: ["icebeam", "hiddenpowerground", "shadowball", "batonpass", "wish"],
+		randomBattleMoves: ["icebeam", "hiddenpowerground", "hiddenpowerfire", "shadowball", "batonpass", "wish", "substitute", "toxic", "protect"],
```

Comment:
I don't think it would be bad. Probably not quite as good as Specs, but it's fat enough to stall out Water-types with Toxic+Wish+Protect that it wouldn't beat with its attacks.